src/game/boe.specials.cpp

```diff
@@ -2812,7 +2812,13 @@ void affect_spec(const runtime_state& ctx) {
 				for(short i = 0; i < 6; i++)
 					if(pc_num == 6 || pc_num == i) {
 						if(spec.ex1b == 0) {
-							if(spec.ex1a == 3 && is_combat() && which_combat_type == 0 && univ.party[i].main_status == eMainStatus::FLED)
+							if(spec.ex1a == 0 && univ.party[i].main_status == eMainStatus::DEAD)
```

Comment:
So, at first glance this seems to be unnecessary. The node already has a check to restore PCs that were dead, dust, or stoned – it's the final `else` in this chain. This change means you can only remove one of the three at a time, and that final fallback will never be hit unless `ex1a` is -1 or something, right?

When you wrote this, was it because raising dead with this node wasn't working at all? Because, if that's the case, that's a different fix from this.

On the other hand, if you've done this because you want a way to unstone PCs without also resurrecting dead PCs, then it seems like a reasonable thing to support.

Reply:
In specials-text-affect.txt, you have:
```
Kill/Raise Dead
Unused
Unused
First part of message
Second part of message
Unused
Unused
Unused
0 - dead, 1 - dust,  2- stoned
0 - raise dead, 1 - hurt
Unused
Unused
Unused
Unused
```
which seems to suggest that if ex1a=0, we only want to raise the dead, if ex1a=1 the dust, ... So we need to add these cases. 

**Note** when converting a legacy game, we need to set ex1a to -1, so it will resurrect all people.

Reply:
That _does_ sound logical, but according to [the documentation](http://openboe.com/docs/editor/appendix/Specials.html?jumpto=64), Extra 1a is ignored when Extra 1b is non-zero, though I think it's a mistake and it meant to say when Extra 1b is _zero_.

As you say, if we make this change, Extra 1a would need to be set to -1 or similar in a legacy game so that it hits the fallback that cures dead, dust, _and_ stone. Also, the HTML documentation would need to be updated.